support-frontend/assets/helpers/redux/checkout/product/contributionsSideEffects.ts

```diff
@@ -5,7 +5,7 @@ import type { ContributionsStartListening } from 'helpers/redux/contributionsSto
 import * as storage from 'helpers/storage/storage';
 import { trackComponentClick } from 'helpers/tracking/behaviour';
 import { sendEventContributionCartValue } from 'helpers/tracking/quantumMetric';
-import { inThreeTierV2Variant } from 'pages/supporter-plus-landing/setup/threeTierABTest';
+import { inThreeTierV3 } from 'pages/supporter-plus-landing/setup/threeTierABTest';
```

Comment:
I wonder whether we should change the name of this so we don't have to update it with every test, we use it to determine whether to show the 3-tier checkout experience if it returns `true`, so maybe we call it `showThreeTierCheckout`?

Reply:
Great spot, renaming was getting laborious. Put it in now. Thanks.